src/utils/assets.ts

```diff
@@ -22,7 +22,9 @@ export interface AssetsMap {
 export const ASSETS_EXTENSION = 'svg';
 
 export const loadPaths = async (dir: string): Promise<string[]> => {
-  const globPath = join(dir, `**/*.${ASSETS_EXTENSION}`);
+  // glob has issues with windows separators, so we need to replace them
+  const globPath = join(dir, `**/*.${ASSETS_EXTENSION}`).replace(/\\/g,'/');
+
   const files = await glob(globPath, {});
```

Comment:
This should be an alternative to the replacement:
```suggestion
  const files = await glob(globPath, { windowsPathsNoEscape: true });
```

Not sure which is better, though.

If doing it that way, the `join` should probably not use a specific separator for consistency:
```js
join(dir, '**', `*.${ASSETS_EXTENSION}`)
```